core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -1106,23 +1106,20 @@ private void parseFromClause(GenericLexer lexer, QueryModel model, QueryModel ma
 
                 if (isCalendarKeyword(tok)) {
                     tok = optTok(lexer);
-
-                    if (tok != null && !isSemicolon(tok)) {
-                        if (isTimeKeyword(tok)) {
-                            expectZone(lexer);
-                            model.setSampleByTimezoneName(expectExpr(lexer));
-                            tok = optTok(lexer);
-
-                            if (tok != null && isWithKeyword(tok)) {
-                                tok = parseWithOffset(lexer, model);
-                            } else {
-                                model.setSampleByOffset(nextConstant("'00:00'"));
-                            }
-                        } else if (isWithKeyword(tok)) {
+                    if (tok == null) {
+                        model.setSampleByTimezoneName(null);
+                        model.setSampleByOffset(nextConstant("'00:00'"));
```

Comment:
this constant can be static

Reply:
actually, it appears I can just pass `null`.